example/test_driver/screens/app_screen.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:test/test.dart';
+
+class AppScreen {
```

Comment:
Why have such class and not have the content inside `test('whatever')` ?

Reply:
can you please explain? @rrousselGit

Reply:
Why is this class existing?

Reply:
`AppScreen` class contains methods and keys of that screen.
and I am calling this method to test cases.

Reply:
The actual tests are in this file `example/test_driver/app_test.dart`

Reply:
Yes but that is not the standard way of writing tests. Why not do it the usual way and have everything in the test file?

Reply:
OK will try this 
give me some time @rrousselGit

Reply:
Done